lit/docs/getting-started/resources.lit

```diff
@@ -0,0 +1,542 @@
+
+  That covers the basics of resources. It is best of think of each resource as
+  a little application, so carefully read any documentation the resource author
+  has provided. Every resource behaves a little differently based on the
+  external system it is representing.
+}
+
+\section{
+  \title{Using External Resource Types}
+  Concourse comes bundled with a lot of resources that are enough for most
+  people to start using Concourse with. However, users will want to extend
+  Concourse to work with all sorts of systems and that means bringing your own
+  \reference{resource-types}.
+
+  Adding a \reference{resource-types}{resource type} to your pipeline looks
+  very similar to adding a resource. You can even override the bundled resource
+  types by re-declaring them in your pipeline.
+
+  Remember, a resource is a container image. So to pull in a new
+  resource type you need to tell Concourse \italic{where} to pull the image
+  from. This is done by using the built-in \link{registry-image
+  resource}{https://github.com/concourse/registry-image-resource/}.
+
+  Here is how we would re-declare the \code{time} resource in our
+  \code{hello-world} pipeline. By default the \link{registry-image
+  resource}{https://github.com/concourse/registry-image-resource/} pulls images
+  from \link{Docker Hub}{https://hub.docker.com/}.
+
+  \codeblock{yaml}{{
+  resource_types: # declare the new resource type
+  - name: new-time
+    type: registry-image
+    source:
+      repository: concourse/time-resource
+
+  resources:
+  - name: every-1min
+    type: new-time # use our new resource type
+    source:
+      interval: 1m
+
+  jobs:
+  - name: hello-world-job
+    plan:
+    - get: every-1min
+      trigger: true
+    - task: hello-world-task
+      config:
+        platform: linux
+        image_resource:
+          type: registry-image
+          source:
+            repository: busybox
+        inputs:
+        - name: every-1min
+        run:
+          path: cat
+          args: ["every-1min/timestamp"]
+  }}
+
+  That's how you add external resource types to your pipeline. You could also
+  call the new resource type \code{time} instead of \code{new-time}.
+}
+
+\section{
+  \title{Time to Fly ✈️}{time-to-fly}
+
+  This brings us to the end of the tutorial. You should have a basic
+  understanding about how to read Concourse pipelines and start creating your
+  own. Here are some other parts of the site to help you take off with Concourse:
+
+  \list{
+    \reference{how-to-guides} - Contains tips for writing pipelines and examples of common pipeline workflows
+  }{
+    Check out all the reference documentation:
+    \list{\reference{jobs}}{\reference{tasks}}{\reference{resources}}{\reference{resource-types}}
+  }{
+    \reference{implementing-resource-types}{Implement your own resource type}
+  }{
+    Find other resources at
+    \link{resource-types.concourse-ci.org}{https://resource-types.concourse-ci.org/}
```

Comment:
I think maybe we can call this out more explicitly in the previous section. Glancing at this section makes it seem like it's a summary, so readers may be inclined to skip over this section if they feel they understood the rest (since they might think there's no new information).